config/defaults_template.yaml

```diff
@@ -18,6 +18,10 @@ languages:
   buildImage: "{{ .BuildImagePrefix }}csharp:{{ .Version }}"
   runImage: mcr.microsoft.com/dotnet/runtime:3.1-bullseye-slim
 
+- language: dotnet
```

Comment:
I guess I'd have to update some buildfiles as well, but it is not clear which ones.
Anything else that need to be done after adding the "build" and "runtime" images for dotnet?

Reply:
You should add targets to the top-level Makefile, modeled on csharp-build-images and csharp-push-images.

You can test this by building and deploying the controller to a test cluster, internally we have "benchmarks-dev1" for that.

I will add some documentation on adding a language to make the steps clearer.

Reply:
updated the makefiles PTAL. Came up with a simpler way to test the docker images for dotnet (see my last comment).